File: infrastructureservice/PlatformCI.InfrastructureService.Models/Implementation/EnvironmentInfrastructure.cs
using System;
using System.Collections.Generic;

namespace PlatformCI.InfrastructureService.Models.Implementation
{
    [Serializable]
    public class EnvironmentInfrastructure
    {
        public string Environment { get; set; }
        public IList<IisApplicationPool> IisApplicationPools { get; set; }
        public IList<ScheduledTask> ScheduledTasks { get; set; }
        public IList<WindowsService> WindowsServices { get; set; }
        public IList<IisSiteApplicationMemberPool> SiteApplicationMembers { get; set; }
        public IList<ServerNameType> Servers { get; set; }
    }
}

File: models/inframodel/environmentinfrastructure.go
package inframodel

type EnvironmentInfrastructure struct {
	Environment            string
	IisApplicationPools    []IisApplicationPool
	ScheduledTasks         []ScheduledTask
	WindowsServices        []WindowsService
	SiteApplicationMembers []IisSiteApplicationMemberPool
	Servers                []ServerNameType
}
